staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/admission.go

```diff
@@ -93,23 +72,17 @@ type WebhookSource interface {
 
 // NewGenericAdmissionWebhook returns a generic admission webhook plugin.
 func NewGenericAdmissionWebhook(configFile io.Reader) (*GenericAdmissionWebhook, error) {
-	kubeconfigFile := ""
-	if configFile != nil {
-		// TODO: move this to a versioned configuration file format
-		var config AdmissionConfig
-		d := yaml.NewYAMLOrJSONDecoder(configFile, 4096)
-		err := d.Decode(&config)
-		if err != nil {
-			return nil, err
-		}
-		kubeconfigFile = config.KubeConfigFile
+	kubeconfigFile, err := config.LoadConfig(configFile)
+	if err != nil {
+		return nil, err
 	}
-	authInfoResolver, err := newDefaultAuthenticationInfoResolver(kubeconfigFile)
+
+	authInfoResolver, err := config.NewDefaultAuthenticationInfoResolver(kubeconfigFile)
 	if err != nil {
 		return nil, err
 	}
 
-	cache, err := lru.New(defaultCacheSize)
+	cm, err := config.NewClientManager(authInfoResolver, config.NewDefaultServiceResolver())
```

Comment:
won't these parameters be nil when this line is called?

Reply:
No, these are the default values.